docs/content/latest/explore/linear-scalability/sharding-data.md

```diff
@@ -294,7 +246,7 @@ Let's get started:
 1. Download the YugabyteDB workload generator JAR file (`yb-sample-apps.jar`):
 
     ```sh
-    $ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.6/yb-sample-apps.jar
+    $ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.3/yb-sample-apps.jar
```

Comment:
Did you lower the version number on purpose here?

Reply:
It wasn't working with 1.3.6 for some reason.